src/index.html

```diff
@@ -17,9 +17,21 @@
 
 <body class="layout_frontend">
     <app-root>
+        <script>
```

Comment:
I put it in the `app-root` tag because then it'll disappear from the DOM once the app loads in. It's really part of the loading elements contained in the `app-root` tag, not something that should hang around for the lifetime of the app. (Unless we decided to remove the theme setting logic from `services.module.ts` per the discussion below, then this should move into the `<header>` tags.)

I know that `<script>` should generally be in `<header>` though, let me know if you still think that's the way to go and I'll change it.